guide/how-it-works/ecash/fedimint.md

```diff
@@ -0,0 +1,200 @@
+   modalImage = "/assets/images/guide/how-it-works/ecash/federated-mint-3@2x.jpg"
+   alt-text = "Diagram showing Federation 1 controlled by four guardians, each with a server and a set of keys. Federation 1 includes a user, an ecash icon, and a bitcoin icon connected by dashed lines."
+   width = 800
+   height = 500
+   layout = "full-width"
+%}
+
+Federated Mints are the standard when it comes to security and resilience in the Fedimint protocol. Managed by four or more guardians, each running their own Fedimint server, this setup distributes control and minimizes risk.
+
+#### Pros:
+
+* **Security** - No single guardian can control the mint, reducing the risk of theft or mismanagement.
+
+* **High resilience** - The mint remains operational even if one or more guardians are temporarily offline, ensuring continuous access.
+
+#### Cons:
+
+* **Complex setup** - Coordinating multiple guardians adds steps and cost.
+
+## Role of guardians
+Guardians are participants who collaboratively manage a federation's operations. They secure the federation's funds using multi-sig wallets, validate payments through consensus, and control ecash issuance and redemption.
+
+## Role of gateways
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/ecash/gateways.png"
+   retina = "/assets/images/guide/how-it-works/ecash/gateways@2x.png"
+   mobile = "/assets/images/guide/how-it-works/ecash/gateways.png"
+   mobileRetina = "/assets/images/guide/how-it-works/ecash/gateways@2x.png"
+   alt-text = "Diagram illustrating the role of a Fedimint gateway in connecting Fedimint ecash and bitcoin on the lightning network. On the left, an icon represents Fedimint ecash with a bitcoin symbol, connected by a dotted line to the center image labeled ‘Gateway,’ showing two hands exchanging cash in front of a vintage computer. On the right, another dotted line connects the gateway to an image of lightning bolts, symbolizing bitcoin on the lightning network."
+   caption = "A Fedimint Gateway lets users pay lightning invoices with Fedimint bitcoin or receive Fedimint bitcoin from lightning payments, without converting between the two assets."
+   width = 800
+   height = 508
+   layout = "full-width"
+%}
+
+A [gateway](https://fedimint.org/docs/GettingStarted/TechCompontents#lightning-swaps) is a service that facilitates interactions between the federation and the lightning network. A gateway acts as a bridge, enabling users within the federation to make payments to and receive payments on the lightning network.
+
+**How gateways work**: The gateway accepts bitcoin payments on the lightning network and converts them into the bitcoin-backed ecash tokens used within the federation. It can also convert bitcoin-backed ecash tokens into bitcoin and send them over the lightning network. This is crucial because it allows the users within a federation to interact with the outside bitcoin and lightning network.
+
```

Comment:
Maybe move to the end of the page?

Reply:
My thinking is that since gateways are a core part of a federation, just like Guardians, they should be explained in a similar section. It makes sense to first explain what a federation is, then how it’s controlled, and finally how federations transfer money between each other (gateways). Gateways are already at the bottom of the explainer, and moving them even further down would feel out of place, as they wouldn’t be explained in the context of a federation anymore.